### DISCUSSION TITLE
HERO MAX 360 Live streaming

### DISCUSSION THREAD

**Discussion by matthdonnet** [gopro/labs#629] [Feature Requests]
Hello, 

I don't know if this is hardware related but I think every Max owner would be incredibly happy to be able to share live 360 videos like other brands of action cameras do.

THanks

**Comment by dnewman-gpsw** [comment]
This will have to happen in main firmware, as Livestreaming is a video mode, and Labs can't add new video modes.

**Reply by matthdonnet** [reply]
ok thanks for the quick reply. Do you know where to post the request then?

KR

Matt

**Reply by dnewman-gpsw** [reply]
MAX2 is in the works, so if it is going to happen, maybe for that camera.

Use https://gopro.com/en/us/info/static/feedback and select product for a feature request
